Last-update: 2014-02-02
Author: Dimitri John Ledkov <xnox@ubuntu.com>
Description: Cherrypick patch from NetBSD, to fix FTBFS with bison 3.0.

--- a/plugins/mathview/itex2mml/itex2MML.y
+++ b/plugins/mathview/itex2mml/itex2MML.y
@@ -1,8 +1,10 @@
 /*             itex2MML 1.4.8
  *   itex2MML.y last modified 9/21/2011
  */
 
+%parse-param {char **ret_str}
+
 %{
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
@@ -26,9 +28,9 @@
    }
 
  void (*itex2MML_error) (const char * msg) = itex2MML_default_error;
 
- static void yyerror (char * s)
+ static void yyerror (char **ret_str, char * s)
    {
      char * msg = itex2MML_copy3 (s, " at token ", yytext);
      if (itex2MML_error)
        (*itex2MML_error) (msg);
